src/components/toast.js

```diff
@@ -191,7 +189,18 @@ const Toast = forwardRef(({
 })
 
 Toast.propTypes = {
-  classes: PropTypes.object,
+  classes: PropTypes.shape({
```

Comment:
Sorry for not seeing this earlier @Hamaad102. I think we can use a shorter form for propTypes here, because everything is string, you can just do this:
```
classes: PropTypes.objectOf(PropTypes.string)
```